cognite/neat/_rules/exporters/_rules2dms.py

```diff
@@ -166,11 +170,16 @@ def export_to_cdf_iterable(
     ) -> Iterable[UploadResult]:
         to_export = self._prepare_exporters(rules, client)
 
+        result_by_name = {}
+        if self.existing_handling == "force":
+            for delete_result in self.delete_from_cdf(rules, client, dry_run, skip_space=True):
+                result_by_name[delete_result.name] = delete_result
```

Comment:
We nee to everything first as that must be done in the reverse order of the creation.